.buildkite/packetbeat/pipeline.packetbeat.yml

```diff
@@ -70,9 +74,9 @@ steps:
           - "packetbeat/build/*.json"
         notify:
           - github_commit_status:
-              context: "packetbeat: Ubuntu Unit Tests"
+              context: "packetbeat: Linux x86_64 Unit Tests"
```

Comment:
I find this change weird, given that we also have a RHEL9 test below which is also Linux.

Can we leave Ubuntu x86_64 instead (or maybe do a sweep of all such references in a future PR and fix them)?

Reply:
I'll create a separate PR and update all contexts there, since there some other places with same syntax